src/sentry/static/sentry/app/views/alerts/wizard/options.tsx

```diff
@@ -0,0 +1,32 @@
+import {t} from 'app/locale';
+
+export type AlertType =
+  | 'issues'
+  | 'num_errors'
+  | 'users_experiencing_errors'
+  | 'throughput'
+  | 'trans_duration'
+  | 'lcp';
+
+export const options: Record<string, [AlertType, string][]> = {
```

Comment:
since we're exporting this, i'd lean towards a less generic name so it doesn't show up in vscode everywhere. Maybe AlertWizardOptions. Same for descriptions

Reply:
ahhh yea that makes sense